### PR TITLE
#144 - expose Liveness and Readiness probes

### PR DESCRIPTION
**Pull request by andytaylor**

### CODE REVIEW

**Comment by andytaylor** [comment]
dont merge this just yet

**Comment by andytaylor** [comment]
Actually its just because of the docs so this can be merged if @brusdev can verify the docs wrt to what the new acceptorname arg will be

File: docs/manual/operator.md
Comment (RoddieKieley) [inline docs/manual/operator.md:264]: Should InitialDelaySeconds and TimeoutSeconds be initialDelaySeconds and timeoutSeconds (lowerCase)?

File: docs/manual/operator.md
Comment (RoddieKieley) [inline docs/manual/operator.md:286]: Should InitialDelaySeconds and TimeoutSeconds be initialDelaySeconds and timeoutSeconds (lowerCase)?

File: docs/manual/operator.md
Comment (RoddieKieley) [inline docs/manual/operator.md:315]: Should InitialDelaySeconds and TimeoutSeconds be initialDelaySeconds and timeoutSeconds (lowerCase)?

**Review by RoddieKieley — COMMENTED** [review]
I fetched, checked out, built, and deployed the operator as built by your branch. I saw that the operator started and that the liveness and readiness probe values were as expected.

However when I updated the livenessProbe and readinessProbe sections from {} to include the initialDelaySeconds: 32 (instead of the default 30) I expected to see the values reflected both in the CR upon reload in the openshift interface as well as see the values reflected in both the container yaml in the pod spec as well as the openshift interface when looking at the details for the container in the pod.

Neither had updated values, but I haven't tested enough to know why this might be the case. Did value updates work for an already deployed broker cluster in either kubernetes or openshift for you?

File: docs/manual/operator.md
Comment (andytaylor) [reply docs/manual/operator.md]: yes I have fixed that

File: docs/manual/operator.md
Comment (andytaylor) [reply docs/manual/operator.md]: also fixed

File: docs/manual/operator.md
Comment (andytaylor) [reply docs/manual/operator.md]: fixed

File: docs/manual/operator.md
Comment (brusdev) [inline docs/manual/operator.md:292]: The option name to set the acceptor name is `--acceptor`, see https://github.com/apache/activemq-artemis/pull/3962

**Review by brusdev — DISMISSED** [review]

File: docs/manual/operator.md
Comment (andytaylor) [reply docs/manual/operator.md]: fixed

**Comment by andytaylor** [comment]
@RoddieKieley fixed th eupdate issue and added a test.

This is good to go now

File: docs/manual/operator.md
Comment (gaohoward) [inline docs/manual/operator.md:277]: typo?

File: controllers/activemqartemis_reconciler.go
Comment (gaohoward) [reply controllers/activemqartemis_reconciler.go:226]: I understand now. It's used to save the custome resource before the NewPodTemplateSpecForCR(fsm) which causes all pods to restart and a new reconcile.

**Pull request merged**
